CP-CPS.md

```diff
@@ -771,6 +773,8 @@ See Section 6.1.
 
 ISRG has created and maintains incident response procedures for a range of potential compromise and disaster situations. Such situations include, but are not limited to, natural disasters, security incidents, and equipment failure. Incident response plans are reviewed, potentially updated, and tested on at least an annual basis.
 
+ISRG maintains a comprehensive and actionable plan for mass revocation events, performs annual testing of its procedures, and incorporates lessons learned to improve preparedness over time.
```

Comment:
This paragraph will be added by https://github.com/letsencrypt/cp-cps/pull/273 when our MRP is complete. Let's remove it and land the other change in this PR independently.
```suggestion
```